api/common.go

```diff
@@ -102,6 +108,8 @@ func FormGroup(key string, start, last int) string {
 	return fmt.Sprintf("%s/%s", group, groupType)
 }
 
+// MatchesContentType validates http media type
```

Comment:
Maybe "MatchesContentType validates the content type against the expected one." ?